.vscode/tasks.json

```diff
@@ -223,6 +264,58 @@
       }
     },
     // ****************
+    // Kernel (python)
+    // ****************
+    // Test
+    {
+      "label": "test (Semantic-Kernel-Python)",
+      "command": "poetry",
+      "type": "shell",
+      "args": ["run", "pytest", "tests/unit"],
+      "problemMatcher": "$msCompile",
+      "group": "test",
+      "presentation": {
+        "reveal": "always",
+        "panel": "shared",
+        "group": "PR-Validate"
+      },
+      "options": {
+        "cwd": "${workspaceFolder}/python"
+      }
+    },
+    {
+      "label": "test (Semantic-Kernel-Python Integration)",
+      "command": "poetry",
+      "type": "shell",
+      "args": ["run", "pytest", "tests/integration","-k","${input:filter}",],
+      "problemMatcher": "$msCompile",
+      "group": "test",
+      "presentation": {
+        "reveal": "always",
+        "panel": "shared",
+        "group": "PR-Validate"
+      },
+      "options": {
+        "cwd": "${workspaceFolder}/python"
+      }
+    },
+    {
+      "label": "test (Semantic-Kernel-Python ALL)",
+      "command": "poetry",
+      "type": "shell",
+      "args": ["run", "pytest", "tests","-k","${input:filter}",],
+      "problemMatcher": "$msCompile",
+      "group": "test",
+      "presentation": {
+        "reveal": "always",
+        "panel": "shared",
+        "group": "PR-Validate"
+      },
+      "options": {
+        "cwd": "${workspaceFolder}/python"
+      }
+    }
```

Comment:
VSCode shows me that comma is required here :)
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/713e2086-bec6-46be-94ac-8788dea9acc6)
```suggestion
    },
```